locustempus/templates/main/course_detail.html

```diff
@@ -15,13 +15,13 @@
             <div class="lt-menu lt-menu-overflow--bootstrap dropdown-menu">
                 <ul class="lt-list">
                     <li class="lt-list-item">
-                        <a href="{% url 'course-edit-view' object.pk %}" class="d-flex align-items-center justify-content-start lt-list-item__link">
+                        <a href="{% url 'course-edit-view' course.pk %}" class="d-flex align-items-center justify-content-start lt-list-item__link">
```

Comment:
I like the change from `object` to `course`. Though the `object` syntax is default Django, I think it is nicer to name the thing you're dealing with.